Formula/chrome-cli.rb

```diff
@@ -1,8 +1,8 @@
 class ChromeCli < Formula
   desc "Control Google Chrome from the command-line"
   homepage "https://github.com/prasmussen/chrome-cli"
-  url "https://github.com/prasmussen/chrome-cli/archive/1.6.0.tar.gz"
-  sha256 "ff1fba560743cba7b661e8daef52d4494acc084da4a21c3fad211f7cdf5e971f"
+  url "https://github.com/prasmussen/chrome-cli/archive/refs/tags/1.7.1.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/prasmussen/chrome-cli/archive/1.7.1.tar.gz"
```
The existing URL format is simpler and resolves to the same tarball download.